tdrs-frontend/src/components/Home/Home.jsx

```diff
@@ -0,0 +1,224 @@
+import React, { useRef, useState } from 'react'
+import Button from '../Button'
+import { faSignOutAlt } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { useDispatch, useSelector } from 'react-redux'
+import signOut from '../../utils/signOut'
+import FormGroup from '../FormGroup'
+import STTComboBox from '../STTComboBox'
+import { requestAccess } from '../../actions/requestAccess'
+
+/**
+ *
+ * @param {string} fieldName - The name of the element that is being validated
+ * @param {string || object} fieldValue - The value of the field. A string from
+ * First Name and Last Name.
+ * The STT fieldValue is either a string or an object if there is a selected
+ * STT.
+ */
+export const validation = (fieldName, fieldValue) => {
+  let field
+  switch (fieldName) {
+    case 'firstName':
+      field = 'First Name'
+      break
+    case 'lastName':
+      field = 'Last Name'
+      break
+    case 'stt':
+      field = 'A state, tribe, or territory'
+      break
+    default:
+      field = ''
+  }
+  if (typeof fieldValue === 'string' && fieldValue.trim() === '') {
+    return `${field} is required`
+  }
+  return null
+}
+
+/**
+ * Home renders the Request Access form for creating a profile, and displays
+ * a pending-approval state, before showing the user their active roles and
+ * permissions once they are approved by an Admin in the backend.
+ */
+function Home() {
```

Comment:
This file is mostly the same as the old Profile page, with additional states handling the transitions based on the access request status.